src/RiakClient/Models/MapReduce/KeyFilters/And.cs

```diff
@@ -13,64 +15,90 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-
-using Newtonsoft.Json;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Text;
+// </copyright>
 
 namespace RiakClient.Models.MapReduce.KeyFilters
 {
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Text;
+    using Newtonsoft.Json;
+
     /// <summary>
     /// Joins two or more key-filter operations with a logical AND operation.
     /// </summary>
     internal class And : IRiakKeyFilterToken
     {
-        private readonly Tuple<string, List<IRiakKeyFilterToken>, List<IRiakKeyFilterToken>> _kfDefinition;
+        /// <summary>
+        ///  Collection of key filter definitions.
+        /// </summary>
+        private readonly Tuple<string, List<IRiakKeyFilterToken>, List<IRiakKeyFilterToken>> keyFilterDefinition;
 
-        public string FunctionName
+        /// <summary>
+        /// Initializes a new instance of the <see cref="And"/> class.
+        /// </summary>
+        /// <param name="left">left hand token</param>
+        /// <param name="right">right hand token</param>
```

Comment:
I suspect if we leave the documentation rules in place at this time, we'll go insane. Maybe we should enable them once we get close to the first official release.